client/src/components/LoginGate.module.css

```
@@ -0,0 +1,120 @@
+    border-radius: 1rem;
+    background: rgba(30, 41, 59, 0.7);
+    backdrop-filter: blur(16px);
+    border: 1px solid rgba(148, 163, 184, 0.12);
+    box-shadow:
+        0 0 0 1px rgba(148, 163, 184, 0.05),
+        0 20px 40px rgba(0, 0, 0, 0.4);
+    text-align: center;
+    animation: fadeIn 0.5s ease-out;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)